net/acme/files/run.sh

```diff
@@ -210,6 +210,7 @@ issue_cert()
 	local user_cleanup
 	local ret
 	local domain_dir
+	local directory_url
```

Comment:
What does "directory_url" mean? Why not just "acme_server" or "server_url"?

Reply:
The resource an ACME client connects to is called "Directory" (see [RFC 8555](https://tools.ietf.org/html/rfc8555#page-23)). Most implementations put that resource at `/directory` (such as Let's Encrypt, which uses the following directory URL: https://acme-v02.api.letsencrypt.org**/directory**)

I have implemented both client and server versions of the RFC and might have a bit of a tunnel vision on the naming here. I think changing it to `acme_server` actually makes more sense and is a very concise name.